etc/publish.sh

```diff
@@ -1,9 +1,6 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-bucket_name=${1:-"sf-fx-nodejs-internal-early-access"}
-aws_cli_profile=${2:-"sf-fx-ea"}
-
 # The package filename cannot be customized for npm pack.
 # We need to figure out the file it will write to on our own:
 name=$(jq -r '.name | gsub("@";"") | gsub("/";"-")' package.json)
```

Comment:
I think this section (up to and including the conditional) can be deleted too now, since  there is no manual tarball being created.

Reply:
done! thanks